### PR TITLE
FIX: MRIQC and datalad

### LINKED ISSUE #352: Add derivatives as subdataset at the end rather than in the beginning
We probably want to have a separate datalad dataset for the derivatives, and only when we are ready to release derivatives, add it to the main dataset as a subdataset, but adding it in the beginning will likely cause problems.
We thus have to modify the SOP to reflect this change

### PR DESCRIPTION
**Pull request by celprov**
Do we have to create a RIA store and a Github siblings for the MRIQC derivatives dataset?
Or can we just push it to the RIA store of the unprocessed data once it has been added as a subdataset?

Closes #352, #351

### CODE REVIEW

File: docs/data-management/mriqc.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 ## Executing *MRIQC*
 
+- [ ] Create a dataset to host the *MRIQC* derivatives.
+    Remember to set the correct version of the container (in our case {{ settings.versions.mriqc }}).
+    ``` shell
+    cd /data/datasets/hcph-dataset
+    datalad create derivatives/mriqc-{{ settings.versions.mriqc }}
```
Comment (oesteban) [inline docs/data-management/mriqc.md]:
```suggestion
    datalad create -c bids mriqc_{{ settings.versions.mriqc }}
```

File: docs/data-management/mriqc.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 ## Executing *MRIQC*
 
+- [ ] Create a dataset to host the *MRIQC* derivatives.
+    Remember to set the correct version of the container (in our case {{ settings.versions.mriqc }}).
+    ``` shell
+    cd /data/datasets/hcph-dataset
```
Comment (oesteban) [inline docs/data-management/mriqc.md]:
```suggestion
    mkdir -p /data/datasets/hcph-derivatives
    cd /data/datasets/hcph-derivatives
```

**Review by oesteban — COMMENTED** [review]
LGTM

**Comment by celprov** [comment]
> LGTM

@oesteban Can you approve the changes in your review? Merge is blocked for now.

**Review by oesteban — APPROVED** [review]

**Pull request merged**